CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ please see [changelog_updates.md](docs/dev/changelog_updates.md).
 - Fix provisioning CaaS breaking dashboard ([#27](https://github.com/sovity/authority-portal/issues/27))
 - Fixed the text being cut off in the connector self-hosting instructions ([#35](https://github.com/sovity/authority-portal/issues/35))
 - Fixed inconsistent placeholder values for privacy policy and legal notice links ([#41](https://github.com/sovity/authority-portal/issues/41))
+- Changed the portal's tab icon and title in browser ([[#48](https://github.com/sovity/authority-portal/issues/48)])
```

Comment:
This is called the "favicon".

I suggest:

```md
- Fixed favicon
- Fixed page titles not aligned with breadcrumb
```

Reply:
thx, done :)